### DISCUSSION TITLE
Items

### DISCUSSION THREAD

**Discussion by deltasolidsnake** [WizardMaster38/Infernum-Ad-Infinitum-To-Hell-and-Beyond#14] [Ideas]
Each comment will be an individual item.

**Comment by deltasolidsnake** [comment]
Name:  Old war flintlock pistol
Description:  A restored flintlock pistol from the war. Slow to reload, but she packs a punch!
Level Needed: 20
Type: USABLE
Rarity: Super rare
Attack: (53 - 89)

**Reply by WizardMaster38** [reply]
Attack?

**Reply by deltasolidsnake** [reply]
on it

**Comment by deltasolidsnake** [comment]
Name:  Hunters slingshot
Description: A slingshot for hunting small game.
Level Needed: 5
Type: USABLE.   
Rarity: Uncommon
Attack: (11 - 23)

**Comment by WizardMaster38** [comment]
Rarities: Common, Uncommon, Rare, Super Rare, Legendary, Mythical
Types: Usable, Consumable, Wearable, Throwable

**Comment by deltasolidsnake** [comment]
Name: Old teddy bear
Description: An old, worn out teddy bear. It's warm to the touch.
Level Needed: 25
Type: Consumable: (Gives the character the ability to resurrect themselves, consumed once)
Rarity: Legendary

**Comment by deltasolidsnake** [comment]
Name: Bokin
Description: A wooden training sword.
Level Needed: 1
Type: useable
Rarity: common
Attack: (3 - 8)

**Comment by deltasolidsnake** [comment]
Name: Iron chair
Description: "Benson is gonna drop his balls when he sees this!"
Level Needed: 18
Type: Useable
Rarity: Rare
Attack: (27 - 38)

**Reply by WizardMaster38** [reply]
THE DESCRIPTION BAHAHA

**Comment by deltasolidsnake** [comment]
Name: Slayer's chest plate
Description: "Rip and Tear"
Level Needed: 50
Type: Wearable
Rarity: Mythic
Defense: (77%)
Bonus: the wearer can enter rage mode, doing 2x damage for two turns.

**Comment by deltasolidsnake** [comment]
Name:  Tome of greed
Description: A book with many fortune spells.
Level Needed: 27
Type: Useable
Rarity: Rare
Bonus: Mages that have this item gain twice the amount of gold on kill.

**Comment by deltasolidsnake** [comment]
Name: Cat ears ad tail
Description: "You dont want to know how you attach the tail.”
Level Needed: 69
Type: Wearable (25 - 30%)
Rarity: Super rare
Bonus: User gets a speed buff.

**Reply by WizardMaster38** [reply]
yes.

**Comment by deltasolidsnake** [comment]
Name: Marksman's crossbow
Description: "Old painless is waiting."
Level Needed: 15
Type: useable 
Rarity: rare
Attack (20 - 32)

**Comment by deltasolidsnake** [comment]
Name: Ranger's boots
Description: A pair of well crafted, but worn boots.
Level Needed: 3
Type: Wearable (12%)
Rarity: Common

**Reply by deltasolidsnake** [reply]
Updated

**Comment by deltasolidsnake** [comment]
Name: Maid outfit
Description: A finely crafted maid outfit.  You're going to put it on the healer... Right?
Level Needed: 18
Type: Wearable (25 - 35%)
Rarity: Rare

**Reply by WizardMaster38** [reply]
R-right..?

**Comment by deltasolidsnake** [comment]
Name: Dwarf amulet
Description: A magic Dwarven amulet.
Level Needed:12
Type: Wearable (Gives the user a shield after being downed, you have one turn to revive them.)
Rarity: Uncommon
